src/component/WordsPerMinute.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+function WordsPerMinute(props) {
+  return (
+    <div className="WPM">
+      {"WPM: " + Math.round(props.textLength / (props.finishTime / 60))}
```

Comment:
Would prefer this was defined in a constant to keep the markup clean. This is fine for now, but this type of thing can very quickly get out of hand.

```
function WordsPerMinute(props) {
  const wpm = Math.round(props.textLength / (props.finishTime / 60));

  return <div class_name="wpm">{wpm}</div>;
}
```